package.json

```diff
@@ -17,6 +18,10 @@
         "test:unit:frontend": "vitest run --config ./config/vitest.config.ts test/unit/frontend/**/*.spec.js",
         "test:unit:frontend-watch": "vitest --config ./config/vitest.config.ts test/unit/frontend/**/*.spec.js",
         "test:system": "mocha 'test/system/**/*_spec.js' --timeout 5000",
+        "test:e2e": "mocha test/system/**/*_spec.js",
+        "test:e2e:frontend": "start-server-and-test serve:test-env http-get://localhost:3000 cy:run",
```

Comment:
Is there a reason the `test:e2e` script was introduced when we already had `test:system` that does exactly the same thing?

We should use a consistent name. We already had `system` in place and is documented - and there is no docs for the `e2e` name currently. I don't mind which way it goes, but we shouldn't have two names for the same thing.

Reply:
Legacy calls that were accidentally left in when trialing different testing frameworks. No longer required and will remove with a PR

Reply:
see: https://github.com/flowforge/flowforge/pull/869